tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -80,3 +80,13 @@ def test_gets_domains_with_cc_and_bcc(self):
         domains = email_parser.get_domains(email)
 
         self.assertSetEqual(domains, {'bar.com', 'com'})
+
+
+class ConvertImgUrlToBase64(TestCase):
+    def test_inlines_image(self):
```

Comment:
This unit test will fail if there's no network connectivity since it makes a live HTTP call. Unit tests shouldn't rely on the network since this is slow and unreliable, e.g. the test might break without a code change because the target URL changed.

You can add a test image into the `tests/files/opwen_email_server/test_email_parser` folder and then mock the requests library to return the bytes of that image. Search for `TEST_DATA_DIRECTORY` to see other tests that use files in that directory.

Reply:
Any advice on how to best approach this? I did a bit of research and I don't find a generally preferred way to mock a `requests.get(url)` with a request to a local file ~ how can I test the url in this scenario? And how can I mock requests.get() if it doesn't accept local file paths? I found this [solution](https://stackoverflow.com/questions/10123929/python-requests-fetch-a-file-from-a-local-url) but I am not 100% convinced about it.